src/screens/ChannelListScreenV2/IncognitoEmptyChat/index.tsx

```diff
@@ -31,9 +32,10 @@ const IncognitoButton = ({title, subtitle, onPress}) => {
   );
 };
 
-const IncognitoEmptyChat = ({totalChannel}) => {
+const IncognitoEmptyChat = () => {
   const {goToContactScreen} = useAnonymousChannelListScreenHook();
   const navigation = useNavigation();
+  const totalChannel = parseInt(StorageUtils.totalAnonChannels.get() || '0', 10);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from passing `totalChannel` as a prop to fetching it from local storage using `StorageUtils` is a good move for decoupling components. However, the parsing of the stored value should be handled inside `StorageUtils` to maintain encapsulation and consistency. This way, if the storage format changes in the future, you only need to update `StorageUtils`, not every usage of it.

```diff
- const totalChannel = parseInt(StorageUtils.totalAnonChannels.get() || '0', 10);
+ const totalChannel = StorageUtils.totalAnonChannels.get();
```

And then in `StorageUtils`:

```diff
- totalAnonChannels: new StorageItem(StorageKeysEnum.TotalAnonChannels),
+ totalAnonChannels: new StorageItem(StorageKeysEnum.TotalAnonChannels, 'number'),
```

Assuming `StorageItem` can take a second argument that specifies the type of the stored value. If it doesn't currently support this, consider adding such functionality.


<!-- This is an auto-generated comment by OSS CodeRabbit -->